src/index.html

```diff
@@ -16,7 +16,92 @@
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="logo-link"
+        target="_blank"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo icon"
+          width="40px"
+          height="40px"
+        />
+      </a>
+
+      <nav>
+        <ul class="nav-list">
+          <li>
+            <a
+              class="is-active"
+              href="#"
+              target="_blank"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              target="_blank"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              target="_blank"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              target="_blank"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              target="_blank"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              target="_blank"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li>
```